test/unit/server_test.js

```diff
@@ -184,6 +184,34 @@ tap.test('createServer:', t => {
     })
   })
 
+  t.test('it should do HTTP content-negotation for *plotly-graph* component', t => {
```

Comment:
What happened to your other content-negotiation tests? Don't you want to test a valid content negotiation?

Reply:
Sorry for the oversight! Thank you so much for the spotting this @etpinard. Removing the tests in my previous commit was a mistake.

Fixed in a95efcf